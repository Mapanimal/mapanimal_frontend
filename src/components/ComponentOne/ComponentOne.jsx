import React from 'react';


const ComponentOne = () => {
    <div className='something'>
     Hi there! I'm a component!
    </div>
}

export default ComponentOne;