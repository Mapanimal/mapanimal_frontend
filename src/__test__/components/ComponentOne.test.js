import React from "react";
import { shallow } from 'enzyme';
import '@testing-library/jest-dom'
import ComponentOne from "../../components/ComponentOne/ComponentOne";
import {createSerializer} from 'enzyme-to-json';

import Enzyme from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';

Enzyme.configure({ adapter: new Adapter() });
expect.addSnapshotSerializer(createSerializer({mode:'deep'}))



describe('tests in <ComponentOne />', () => {
      test('<ComponentOne /> it renders well', () => {
        const title = "Hi there! I'm a component!";
        const componentOne = shallow(<ComponentOne title={title} />);
        expect(componentOne).toMatchSnapshot();
      });
    });