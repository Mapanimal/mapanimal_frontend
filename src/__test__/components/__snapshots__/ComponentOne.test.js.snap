// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests in <ComponentOne /> <ComponentOne /> it renders well 1`] = `""`;
